Task1/Project/Shapes/Models/Classes/Circle.cs

```diff
@@ -38,7 +39,7 @@ public Circle(Point сenter, double radius)
         /// <summary>
         /// Identifier of the circle.
         /// </summary>
-        public override string ID => nameof(Circle);
+        public override string ID => ShapeBase.IdOfTheShape(this.GetType());

```

Comment:
you do not need ID in all child classes, just write it in base class once, that would be enough

Reply:
Ok.